docs/dapp/opl/README.mdx

```diff
@@ -0,0 +1,79 @@
+---
+description: Integrating privacy into dApps on other blockchains
+---
+
+import DocCard from '@theme/DocCard';
+import DocCardList from '@theme/DocCardList';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# Oasis Privacy Layer (OPL)
+
+The Oasis Privacy Layer (OPL) is a powerful solution that enables developers
+to integrate privacy features into their decentralized applications (dApps)
+across multiple EVM-compatible networks. 
+
+- **Privacy-First**: OPL leverages the [Sapphire]'s privacy features to ensure
+that contract data and computation remains confidential.
+- **Cross-Chain Compatibility**: OPL is compatible with multiple blockchains
+through message bridging protocols, making it easy to integrate privacy
+regardless of the chain your dApp is built on.
+
+For more information about OPL and to catch the latest news, please visit the
+[official OPL page].
+
+[official OPL page]: https://oasisprotocol.org/opl
+[Sapphire]: https://oasisprotocol.org/sapphire
+
+## How OPL Works
+
+The OPL is made possible through message bridges, which enable secure
+communication between OPL-enabled contracts on Sapphire and smart contracts on
+other chains. This allows dApps to access privacy-preserving capabilities while
+keeping their main logic on their primary chain.
+
+![Oasis Privacy Layer diagram](../images/opl/privacy-layer-diagram.png)
+
+:::info
+
+For how to use use signed messages with the GSN to trigger a cross-chain
+messages, please visit our [Gasless Transactions chapter].
+
+:::
+
+[Gasless Transactions chapter]: ../sapphire/gasless.md
+
+## Message Bridges
+
+You can integrate message bridges into your dApps using one of these three
+methods:
+
+- **OPL-SDK**: A wrapper provided by the Oasis Protocol that simplifies the
+integration of message bridging with Oasis’s privacy features.
+- **Celer Inter-Chain Messaging (IM)**: A generalized message bridging solution
+by Celer, which lets you build more complex solutions.
+- **Router Protocol CrossTalk**: An extensible cross-chain framework that
+enables seamless state transitions across multiple chains.
+
+### Comparison
+
+| Protocol        | Validator Network | Relayer | Fees |
+| --------------- | ----------------- | ------ | ------ |
+| **OPL-SDK**         | SGN (Celer)                       | Executor (self-hosted or hosted service by Celer) | SGN Fee: Paid via `msg.value` <br/> Executor Fee: Carged externally (Free on testnet) |
+| **Celer-IM**        | SGN (Celer)                       | Executor (self-hosted or hosted service by Celer) | SGN Fee: Paid via `msg.value` <br/> Executor Fee: Carged externally (Free on testnet) |
+| **Router Protocol** | Orchestrators (Router Chain)      | Relayer (run by 3rd party) | Paid by the approved feepayer on the Routerchain |
+
+## See also 
+<DocCardList items={[
+    findSidebarItem('/dapp/opl/opl-sdk/'),
+    findSidebarItem('/dapp/opl/celer/'),
+    findSidebarItem('/dapp/opl/router-protocol/')
+    ]} />
+
```

Comment:
The See also section should reference related chapters from other parts of the docs and not subchapters.

```suggestion
```